### PR TITLE
Add ability to get raw Pokémon filtered by id

### PR DESCRIPTION
**Pull request by fettlaus**
Added a parameter for filtering specific Pokémon.

## Description
Added a paramter to the `/raw_data` endpoint for requesting only selected and active Pokémon by their id. Just add a parameter with comma separated values to the request like this: `http://localhost:5000/raw_data?gyms=false&scanned=false&ids=13,16` and the server will only respond with active pidgeys and weedles.

## Motivation and Context
Using the raw data as an endpoint for further development for example as input to a mobile application, we need some way to filter on the server side. Especially regarding mobile data usage.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: pogom/models.py [hunk 1]
```diff
@@ -55,6 +55,21 @@ def get_active(cls):
 
         return pokemons
 
+    @classmethod
+    def get_active_by_id(cls, ids):
+        query = (Pokemon
+                 .select()
+                 .where((Pokemon.pokemon_id << ids) &
```
Comment (MonkeyLeeT) [inline pogom/models.py]: Shifting operator with list? How does this work?

Comment (fettlaus) [inline pogom/models.py]: It's a function provided by peewee replacing the  pythonic `in` operator.

Am 23.07.2016 21:46 schrieb "Ted Li" <notifications@github.com>:

Comment (MonkeyLeeT) [inline pogom/models.py]: Okay, hopefully no one in this project will wanna use shift operator...

**Comment by craywong** [comment]
Is it just me that this feature is not working?

**Comment by fettlaus** [comment]
What was the URL you called?

**Pull request merged**